app/indexers/essi/file_set_indexer.rb

```diff
@@ -6,8 +6,11 @@ class FileSetIndexer < Hyrax::FileSetIndexer
 
     def generate_solr_document
       super.tap do |solr_doc|
-        solr_doc['is_page_of_ssi'] = object.parent.id if object.parent
```

Comment:
For reviewers:  This series of calls to the parent resulted in four LDP load operations: once for each parent method invoked but _also_ for each safety check in the `if` clause.